app/screens/search/search.js

```diff
@@ -514,43 +516,53 @@ export default class Search extends PureComponent {
             value,
         } = this.state;
         const style = getStyleFromTheme(theme);
-        const sections = [{
-            data: [{
-                value: 'from:',
-                modifier: `from:${intl.formatMessage({id: 'mobile.search.from_modifier_title', defaultMessage: 'username'})}`,
-                description: intl.formatMessage({
-                    id: 'mobile.search.from_modifier_description',
-                    defaultMessage: 'to find posts from specific users',
-                }),
-            }, {
-                value: 'in:',
-                modifier: `in:${intl.formatMessage({id: 'mobile.search.in_modifier_title', defaultMessage: 'channel-name'})}`,
-                description: intl.formatMessage({
-                    id: 'mobile.search.in_modifier_description',
-                    defaultMessage: 'to find posts in specific channels',
-                }),
-            }, {
+        
+        let sectionsData = [{
+            value: 'from:',
+            modifier: `from:${intl.formatMessage({id: 'mobile.search.from_modifier_title', defaultMessage: 'username'})}`,
+            description: intl.formatMessage({
+                id: 'mobile.search.from_modifier_description',
+                defaultMessage: 'to find posts from specific users',
+            }),
+        }, {
+            value: 'in:',
+            modifier: `in:${intl.formatMessage({id: 'mobile.search.in_modifier_title', defaultMessage: 'channel-name'})}`,
+            description: intl.formatMessage({
+                id: 'mobile.search.in_modifier_description',
+                defaultMessage: 'to find posts in specific channels',
+            }),
+        }];
+
+        // if search by date filters supported
+        if (this.props.enableDateSuggestion) {
```

Comment:
not needed

Reply:
Why is this not needed now?

Reply:
that's an outdated comment, it is needed, we spoke via mattermost about it